### ISSUE TITLE
Online discoverDomain does not output the Machine attribute of the server

### ISSUE THREAD

**Issue by knutze** [oracle/weblogic-deploy-tooling#813]
WDT version is 1.9.7 and the WLS version is 12.2.1.4.

createDomain using a model file that was discoverDomain in online mode, the association between the server and the machine seems to be lost.

Comparing the online and offline model files, we can see that the Machine attribute of the server is missing.

```diff
 topology:
     Server:
         'Server-0':
-            Machine: 'Machine-0'
     UnixMachine:
         'Machine-0':
```

Thank you.

**Comment by rakillen** [comment]
We'll keep this issue open for a future release.

Online discover has a number of known problems similar to the two you have found, and often requires manual adjustments to the model. I'd recommend using offline discover if it fits your use case.

The problem you experienced with offline discover in issue #795 has been fixed in the 1.9.8 release of WDT. I tried it out with a 10.3.6 domain, and it ran successfully. There is a warning that it is unable to discover the SecurityConfiguration, but that is only a concern if you have a customized security configuration.
